docs/developer-guide/etendo-classic/how-to-guides/how-to-create-a-computed-column.md

```diff
@@ -0,0 +1,137 @@
+---
+tags: 
+  - Computed column
+  - How to
+  - SQL logic
+  - Etendo classic
+---
+
+# How to Create a Computed Column
+
+## Overview
+
+A computed column is a column of a table which is computed using an SQL expression and which does not exist explicitly in the database schema.
```

Comment:
I think that the SQL reference should be highlighted, such as sqllogic field / whereclause. Everything that makes a references to an AD field that is commonly used.